### PR TITLE
getting_started: add debug information for users/distros

### PR DESCRIPTION
**Pull request by plbossart**
Rather than provide the same guidance over and over in GitHub issues,
write down a set of suggestions to help users/distros solve their
issues or report issues with more information.

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>

### CODE REVIEW

File: getting_started/intel_debug/suggestions.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+   options snd_intel_dspcfg dyndbg=+p
+
+This list is only an example.
+
+Install sof-logger
+******************
+
+If an issue with the SOF firmware is reported, such as IPC errors, SOF
+developers will need DSP traces. This is typically done by installing
+/usr/local/bin/sof-logger as well as the .ldc file, and using the
```
Comment (lgirdwood) [inline getting_started/intel_debug/suggestions.rst:108]: We could include a link here to sof-bin to get all binaries/scripts etc ?

**Review by lgirdwood — COMMENTED** [review]
I'm guessing the bug report template will link to this once upstream ?

File: getting_started/intel_debug/suggestions.rst (hunk 1 above)
Comment (plbossart) [reply getting_started/intel_debug/suggestions.rst:108]: I thought about it but we have an AR to move sof-logger to a separate repo, so not sure if we want to add details and then change again.

**Comment by plbossart** [comment]
> I'm guessing the bug report template will link to this once upstream ?

yes, that was the idea, add a 'before you file, see this link" type of thing.

**Review by lgirdwood — APPROVED** [review]

**Review by deb-intel — APPROVED** [review]
Will make minor grammatical updates in a separate PR.

**Pull request merged**